.github/workflows/yeschoy-deploy.yml

```
@@ -0,0 +1,196 @@
+
+          tags: |
+            ghcr.io/yeschoy/new-api:${{ github.sha }}
+            ghcr.io/yeschoy/new-api:production
+
+          # GitHub Actions Docker layer cache
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+
+      - name: Build summary
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:27: Double quote to prevent globbing and word splitting

(shellcheck)